### DISCUSSION TITLE
Track Hong Kong stock?

### DISCUSSION THREAD

**Discussion by charleslcso** [ghostfolio/ghostfolio#1161] [Q&A] [answered]
I typed in ISIN number for a stock in HK, and it is not accepted. I wonder what to do to make it work. 

The ISIN is from a list downloadable from HKEX. It is Bank of East Asia. 

https://www.hkex.com.hk/-/media/HKEX-Market/Services/Trading/Securities/Securities-Lists/ISINs-assigned-by-HKEX/isinsehk.xls

![image](https://user-images.githubusercontent.com/520537/185075289-f0e51148-bf54-48c8-9948-c9c61efe96af.png)

**Comment by dtslvr** [comment, marked as answer]
Hi @charleslcso

When I search for `HK0023000190`, I get `0023.HK` as a suggestion. Just select it and the field will be valid.

<img width="793" alt="HK0023000190" src="https://user-images.githubusercontent.com/4159106/185112314-77f0e772-64c8-4a88-b7b5-b5dcc1344eb4.png">
